src/Microsoft.Health.Fhir.TemplateManagement/Constants.cs

```diff
@@ -8,7 +8,15 @@ namespace Microsoft.Health.Fhir.TemplateManagement
     internal static class Constants
     {
         // Accept media type for manifest.
-        internal const string MediatypeV2Manifest = "application/vnd.docker.distribution.manifest.v2+json";
+        internal const string MediaTypeManifest = "application/vnd.docker.distribution.manifest.v2+json";
```

Comment:
Can we add some comments to this type, like references?

Reply:
Added, thanks.